types/node/constants.d.ts

```diff
@@ -11,3 +11,8 @@ declare module 'constants' {
         typeof fsConstants;
     export = exp;
 }
+
+declare module 'node:constants' {
+    import * as constants from 'process';
+    export = constants;
```

Comment:
Shouldn't this be importing from `'constants'`? Also, this is mixing `import *` with `export =`, which seems confusing. Since `constants` uses `export =`, I'd recommend using `import =` for consistency here:

```suggestion
    import constants = require('constants');
    export = constants;
```

Reply:
Yea, straight up typo here, which is weird since I automated that bit...